src/extensions/extension-discovery.ts

```diff
@@ -114,7 +114,7 @@ export class ExtensionDiscovery {
     // chokidar works better than fs.watch
     chokidar.watch(this.localFolderPath, {
       // Dont watch recursively into subdirectories
-      depth: 0,
+      depth: 1,
```

Comment:
Could you explain a little bit more (in the PR description) why this fix symlinking? There seems to be some discussion in Slack, but maybe some team member (like me) didn't fully follow.

Reply:
Could add comment to the code explaining that depth: 1 is required for symlinking to work.

Reply:
@nevalla also noticed that depth: 1 is required in Windows

Reply:
Should I add that to the comment as well?

Reply:
Not necessarily I guess, just FYI.